### PR TITLE
chore(commit): using commitizen for smart commit

### LINKED ISSUE #26: Add commitizen for commit conventions
Is that pending on purpose?

### PR DESCRIPTION
**Pull request by sarbbottam**
Closes #26

added `npm run commit` for smart commit messages.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,27 +7,22 @@
     "dist"
   ],
   "scripts": {
+    "commit": "git-cz",
     "prebuild": "rimraf dist",
     "validate": "npm-run-all --parallel lint cover build test:configs --sequential check-coverage",
     "build": "babel -d dist src",
     "check-coverage": "nyc check-coverage --statements 100 --branches 100 --functions 100 --lines 100",
     "lint": "eslint .",
-
```
Comment (sarbbottam) [inline package.json]: This lines have got auto deleted when installing deps with `-D` flag.

Comment (kentcdodds) [inline package.json]: :+1:

**Comment by kentcdodds** [comment]
Lgtm

**Comment by kentcdodds** [comment]
Could we rebase this?

**Comment by sarbbottam** [comment]
Rebased with `upstream/master` and resolved merge conflicts

**Comment by kentcdodds** [comment]
Do you mind adding the `commitizen friendly` badge?

```
[![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg?style=flat-square)](http://commitizen.github.io/cz-cli/)
```

And maybe add a note in `CONTRIBUTING.md`. Sorry I didn't ask you to do this earlier :-/

**Comment by sarbbottam** [comment]
No worries, sorry for the delayed response, Monday :large_blue_circle:

**Comment by kentcdodds** [comment]
Looks great!

**Pull request merged**